public/app/features/explore/ExploreTimeControls.tsx

```diff
@@ -66,19 +69,25 @@ export class ExploreTimeControls extends Component<Props> {
     });
   };
 
+  changeTimeSync = () => {
+    const { onChangeTimeSync, exploreId } = this.props;
+    onChangeTimeSync(exploreId);
```

Comment:
We probably do not need to pass the `exploreId` to the controls at all. We could just accept `onChangeTimeSync: () => void;` and let the parent decide how to, or whether to map `exploreId` in it somehow.